mythril/laser/ethereum/svm.py

```diff
@@ -102,6 +103,18 @@ def __init__(
 
         self.iprof = InstructionProfiler() if enable_iprof else None
 
+        if enable_coverage_strategy:
+            from mythril.laser.ethereum.plugins.implementations.coverage.coverage_plugin import (
+                InstructionCoveragePlugin,
+            )
+            from mythril.laser.ethereum.plugins.implementations.coverage.coverage_strategy import (
+                CoverageStrategy,
+            )
+
+            instruction = InstructionCoveragePlugin()
```

Comment:
InstructionCoveragePlugin is already being initialised by plugin loader, that should be used in here.